infermedica_api/connectors/v2.py

```diff
@@ -0,0 +1,428 @@
+from typing import Optional, List, Dict
+
+from .common import APIConnector, SEARCH_FILTERS
+from .. import exceptions, models
+
+
+class APIv2Connector(APIConnector):
+    """
+    Intermediate level class which handles requests to the Infermedica API,
+    provides methods with detailed parameters, but still works on simple data structures.
+    """
+
+    def __init__(self, *args, api_version='v2', **kwargs: Dict):
```

Comment:
What if `api_version='v3’`? Do we have any validation here? Maybe we should raise expiation in that case?